src/components/Form.jsx

```diff
@@ -0,0 +1,102 @@
+import { InputLabel, TextField } from '@mui/material';
+import BootstrapButton from '../components/Button';
+import { styled } from '@mui/material/styles';
+
+function Form(props) {
+  const label = {
+    fontFamily: 'Open Sans, sans-serif',
+    color: 'black',
+    fontWeight: 'bold',
+    fontSize: '16px',
+    marginTop: '17px'
+  };
+
+  const CssTextField = styled(TextField)({
+    width: '90%',
+    height: '50px',
+    'background-color': '#F5F5F5',
+    'border-radius': '5px',
+    '& label.Mui-focused': {
+      color: props.colorHover
+    },
+    '& .MuiInput-underline:after': {
+      border: 'none',
+      'border-bottom': '2px solid ' + props.colorHover
+    },
+    '& .MuiOutlinedInput-root': {
+      '& fieldset': {
+        border: 'none'
+      },
+      height: '50px',
+      '&:hover fieldset': {
+        border: 'none',
+        'border-bottom': '2px solid ' + props.colorHover
+      },
+      '&.Mui-focused fieldset': {
+        border: 'none',
+        'border-bottom': '2px solid ' + props.colorHover
+      }
+    }
+  });
+
+  return (
+    <div className="">
+      <div className="ContForm">
+        <form id="form1">
+          <div className="textFields1">
+            <InputLabel style={label}>{props.FirstInputTitleText}</InputLabel>
+            <CssTextField
```

Comment:
If Inputs should be styled outise the form component and just allow load childrens inside the form component